doc/classes/InputEvent.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Abstract base class of all types of input events. See [method Node._input].
+		[b]Note:[/b] Custom classes inheriting [InputEvent] will [i]not[/i] propagate automatically. To pass additional data attached to an [InputEvent], use [method Object.set_meta] on the built-in [InputEvent] classes instead.
```

Comment:
Is it worth being specific here? Probably not. There are multiple ways to manually "feed" input events, aren't there?
```suggestion
		[b]Note:[/b] Custom classes inheriting [InputEvent] will [i]not[/i] propagate automatically through [method Input.parse_input_event]. To pass additional data attached to an [InputEvent], use [method Object.set_meta] on the built-in [InputEvent] classes instead.
```

Reply:
my .02: being specific can help less experienced users to interpret the message more clearly, as they may not know that this only applies to parse_input_event(I didn't either)

Reply:
Note that there's also `Viewport.push_input` but my argument still stands.

Reply:
I suppose it's true that you can't put all the relevant documentation in an error message like this. My final piece of feedback then is to alter the documentation to include more references to this information in docs pertaining to similar topics, such as the "using InputEvent" document. It can be hard to work out a good solution if you don't know what information to search for.

Either way thanks for picking this up so quickly, this has been a source of quite some frustration for me but I'm glad to see a fast response to feedback like this.

Reply:
Note that in some cases the metadata is *not* preserved, see:
* #95205